ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/PartitionExport.java

```diff
@@ -75,64 +77,62 @@ class PartitionExport {
     this.callersSession = SessionState.get();
   }
 
-  List<DataCopyPath> write(final ReplicationSpec forReplicationSpec, boolean isExportTask,
+  public void exportPartitions(final ReplicationSpec forReplicationSpec, boolean isExportTask,
+                            FileList fileList, boolean dataCopyAtLoad) throws RuntimeException {
```

Comment:
Please introduce an additional prefix and keep the same base name. Both of backing and wrapping methods share a same logical behavior, but in a different thread model. For example, parallelXxx(...) and xxx(...) can help readers to what they represent. Additionally, it's better to have a Javadoc document to explicitly say that parallelXxx is a parallelized version of xxx. You can use @ see annotation for its reference.

exportPartitions and write may lead readers to think that they do different things, and exportPartitions doesn't give a good clue about thread models.

A same advice applies for exportTable method, too.